plugins/catalog-backend/src/service/DefaultLocationService.ts

```diff
@@ -26,18 +26,26 @@ import { LocationInput, LocationService, LocationStore } from './types';
 import { locationSpecToMetadataName } from '../util/conversion';
 import { InputError } from '@backstage/errors';
 import { DeferredEntity } from '@backstage/plugin-catalog-node';
+import { Config } from '@backstage/config';
 
 export class DefaultLocationService implements LocationService {
   constructor(
     private readonly store: LocationStore,
     private readonly orchestrator: CatalogProcessingOrchestrator,
+    private readonly config: Config,
```

Comment:
Let's make this an options object instead. 

`private readonly options: { allowedLocationTypes: string[] } = { allowedLocationTypes: ['url'] }`